cicd/pipelines/configuring-security-context-for-pods.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="configuring-security-context-for-pods"]
+= Configuring the security context for pods
+include::_attributes/common-attributes.adoc[]
+:context: configuring-security-context-for-pods
+
+toc::[]
+
+The default service account for pods that {pipeline-shortnames} starts is `pipeline`. The security context constraint (SCC) associated with the `pipeline` service account is `pipelines-scc`. The `pipelines-scc` SCC is similar to the `anyuid` SCC, but with minor differences as defined in the following YAML specification:
+
+.Example `pipelines-scc.yaml` snippet
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+...
+allowedCapabilities:
+  - SETFCAP
+...
+fsGroup:
+  type: MustRunAs
+...
+----
+
+In addition, the `Buildah` cluster task, shipped as part of {pipelines-shortname}, uses `vfs` as the default storage driver.
```

Comment:
I am almost wondering if this `Bildah` information should be in a procedure module with more information on how you can make "the following changes" it seems out of place in this assembly, but I am not sure.

Reply:
This information was there since the previous version, so until a deep restructure (which I hope to complete eventually) I would not want to move it.